docs/workflow.rst

```diff
@@ -278,49 +205,23 @@ There are 3 different flavors of observation models.
    physic grid by interpolating between the sparse grid computed from the AST
    results.
 
-Create the AST input list
--------------------------
-
-To be added.
-
-Compute the ASTs
-----------------
-
-Done separately with the same code that was used to extract the source
-photometry.
-
-
-Split up the ASTs by source density
------------------------------------
-
-To be added.
-
-Currently the workflow assumes a single AST file for all the source densities.
-
-Create the observation models for each source density
------------------------------------------------------
-
-To be added.
-
-Create a single observation model
----------------------------------
-
-This assumes that the ASTs do not have a strong dependence on source
-density.  This could be a good approximation if the source density does
-not change much over the observation area or is low everywhere.
-The '0 0' arguments are dummy values.
+The code to compute the observation can be done with or without subgridding, and
+with or without source density splitting.  Here are some examples:
 
   .. code-block:: console
 
-     $ ./run_beast_production.py -o datafile.fits 0 0
+     $ # with source density splitting and no subgridding
+     $ beast.tools.run.create_obsmodel --use_sd --nsubs 1
+     $ # with source density splitting and 5 subgrids
+     $ beast.tools.run.create_obsmodel --use_sd --nsubs 5
+     $ # no source density splitting or subgrids
+     $ beast.tools.run.create_obsmodel --nsubs 1
```

Comment:
Need to covert commands to command line versions (see prev comments).

Reply:
resolved per discussion below

Reply:
resolved per discussion below